cosmo_tester/resources/attributes.yaml

```diff
@@ -4,6 +4,12 @@ cloudify_username: admin
 cloudify_password: admin
 cloudify_tenant: default_tenant
 
+# This should be set to true if testing community images.
+# This isn't detected by the framework, as doing so would require either some
+# heuristics or deploying managers for each test before determining whether it
+# could be skipped, both of which have a good chance of wasting a lot of time.
+image_is_community: false
```

Comment:
Not env vars I think- we went there before and it was painful.

Anyway, if we want to test a community image then the config needs modifying anyway (to set the image name). I think that's more under the remit of the config changes either way?